packages/cmf/src/localStorage.md

```diff
@@ -0,0 +1,22 @@
+# setup localStorage with CMF
+
+You can use this API to setup your redux state using localStorage.
+
+```javascript
+import cmf from '@talend/react-cmf';
+
+const localStorageKey = 'myApp-v1';
+const preloadedState = cmf.localStorage.getState(localStorageKey);
+const storeCallback = cmf.localStorage.getStoreCallback(localStorageKey, [
+    ['cmf', 'components', 'Container(List)', 'foo'],
+    ['cmf', 'components', 'Container(SidePanel)'],
+]);
+
+cmf.bootstrap({
+    preloadedState,
+    storeCallback,
+});
+```
+
+From now the redux-storage api provided in CMF is DEPRECATED.
+It will be removed in the next major release.
```

Comment:
How do you deal with this removal? Another TUI?

Reply:
good point, I have added a subtask for that and update description